test/cqlpy/conftest.py

```diff
@@ -41,13 +41,20 @@ def pytest_addoption(parser):
     add_s3_options(parser)
 
 
+@pytest.fixture(scope=testpy_test_fixture_scope)
+async def host(request, testpy_test: PythonTest | None):
+    if testpy_test is None:
+        yield request.config.getoption("--host")
+    else:
+        async with testpy_test.run_ctx(options=testpy_test.suite.options):
+            yield testpy_test.server_address
```

Comment:
Does this really need to be a async fixture? Currently cqlpy doesn't use async anywhere.

Reply:
This is because `testpy_test.run_ctx()` is async function. Yes, it can be wrapped with `@universalasync.wrap` decorator, but I prefer to limit the usage of this decorator.

Anyway, `pytest_asyncio` module also has some magic inside to make async/sync fixtures work together.